http/grpc/README.md

```diff
@@ -0,0 +1,14 @@
+# Coverage for various GRPC-related things
+
+## Required key generation steps
+
+Based on https://github.com/quarkusio/quarkus-quickstarts/tree/main/grpc-tls-quickstart/src/main/resources/tls
+
+```shell
+    # create ca
+`    openssl req -x509 -new -newkey rsa:2048 -nodes -keyout ca.key -out ca.pem  -config ca.cnf -days 365 -extensions v3_req -subj "/C=CZ/ST=JMK/L=Brno/O=RedHat/OU=Quarkus-QE"
```

Comment:
Does it mean in a year time tests will start failing? I think it could mean we will need to be updating certs in non-main branch. Maybe we should have expiration date beyond RHBQ release lifetime.

Reply:
Good point, fixed